playbooks/pmm2-client_integration_upgrade_custom_path.yml

```diff
@@ -58,34 +26,10 @@
       instance_ip: "{{ instance_ip.stdout }}"
 
   - name: include tasks for test env setup
-    include_tasks: ../tasks/test_prep.yml
-
-  - name: include tasks for enabling main repo
-    include_tasks: ../tasks/enable_main_repo.yml
+    include_tasks: ../tasks/pmm2_client_test_prepare.yml
 
-  - name: install PMM client old deb packages
-    apt: name={{item}} update_cache=yes state=latest
-    with_items:
-    - pmm2-client
-    when: ansible_os_family == "Debian"
-
-  - name: install acl packages
-    apt: name={{item}} update_cache=yes state=latest
-    with_items:
-    - acl
-    when: ansible_os_family == "Debian"
-
-  - name: install PMM client old rpm packages
-    yum: name={{item}} state=latest
-    with_items:
-    - pmm2-client
-    when: ansible_os_family == "RedHat"
-
-  - name: setup pmm2-client artifacts for custom path
-    command: "{{item}}"
-    with_items:
-      - "chmod +x /package-testing/scripts/pmm2_client_custom_path.sh"
-      - "/package-testing/scripts/pmm2_client_custom_path.sh"
+  - name: install old pmm2-client tarball to custom path
+    command: "../scripts/pmm2_client_install_tarball.sh -v 2.26.0 -p {{ custom_path }}"
```

Comment:
why do you specifically install 2.26.0 here?

Reply:
that's a temporary solution while I'm working on obtaining release version. it's fixed now.

Reply:
restored the hardcode until PMM-10247 is fixed

Reply:
this way we will have ability to run upgrade tests to RC version